stacks-signer/src/client/stacks_client.rs

```diff
@@ -383,6 +383,22 @@ impl StacksClient {
         })
     }
 
+    /// Get the current reward cycle and whether the prepare phase has started for the next cycle
+    pub fn get_current_reward_cycle_and_prepare_status(&self) -> Result<(u64, bool), ClientError> {
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Yep, removed in [c50a930](https://github.com/stacks-network/stacks-core/pull/4481/commits/c50a93088d7c0261f1dbe31ab24b95028a038447)